StylusDeepDark.user.css

```diff
@@ -171,12 +171,21 @@
 		--main-text: /*[[mainText]]*/;
 		--dimer-text: /*[[dimerText]]*/;
 
-		--shadow: 0 1px 0.5px rgba(0,0,0,0.13);
+		--shadow: 0 1px 0.5px rgba(0, 0, 0, 0.13);
 
 		/*Cursor width and color*/
 		--cursor-width: /*[[cursorWidth]]*/;
 		--cursor-color: /*[[cursorColor]]*/;
 
+		/*Disable all color*/
```

Comment:
@the-j0k3r Yo, joker. Please restrain yourself from using new colors. My themes are made so that you can change the colors at have it look great. IF you look at the code colors you can see that I took a lot of time to filter the colors to get some more colors out of the 6 I can work with.

#d62929 can be main but inverted
#f33 why a second shade of red?
#d99b45 this is yellow but with some hue rotate we can manage
rgba(0, 0, 0, .16) very very dark gray :man_facepalming: 

So... yeah, FIX THOSE.

Here's what hyperfekt is doing: https://github.com/RaitaroH/google-dark/blob/master/search/search.user.css
`@var text main-border "Main Border" alpha(#31363b, 0.9 * contrast)`

I personally just have 6 variables and play with them in the code later on. Choose what you do. I would rather prefer the later